Arrays/Valid Subsequence/Valid_Subsequence.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Oct 25 18:29:39 2020
+
+@author: andrew
+"""
+
+'import Validate BST from Trees folder (had to add a "_" to call it)\
+    I use Linux, this method works for me; your path call line 13 may be different'
+import sys
+# insert at 1, 0 is the script path (or '' in REPL)
+sys.path.insert(1, '../../Trees/')
```

Comment:
Currently we are creating separate solutions for each problems. That's why we can't accept importing from another solution. You must create a separate solution for Valid_Subsequence

Reply:
Hello. No problem, here goes the new file, which is stand-alone. :)